src/tokenizer/main.py

```diff
@@ -150,6 +213,30 @@ def val(t, quote_word=False):
     else:
         to_text = lambda t: t.txt
 
+    if args.convert_measurements:
+        options["convert_measurements"] = True
+
+    if args.with_annotation:
```

Comment:
I am still not sure that this option makes sense. First, it is definitely True by default when you invoke the Tokenizer from the command line - and therefore the option should reverse it to False, if present. Second, I think this is really an internal option that is only ever going to be used by GreynirPackage (and perhaps other similar packages that integrate tightly with Tokenizer). So I think it can be dropped as a command-line argument.

Reply:
Deleted!